packages/tldraw/src/lib/ui/components/SharePanel/UserPresenceEditor.tsx

```diff
@@ -36,6 +36,12 @@ export function UserPresenceEditor() {
 		rOriginalName.current = rCurrentName.current
 	}, [trackEvent])
 
+	const handleCancel = useCallback(() => {
+		setIsEditingName(false)
+		editor.user.updateUserPreferences({ name: rOriginalName.current })
```

Comment:
Coudn't use marks here since the user preferences are stored in the local storage.